src/tests/topp/PeptideIndexer_2_out.idXML

```diff
@@ -1,55 +1,55 @@
-                <UserParam type="string" name="target_decoy" value="target"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_1" accession="BSA2" score="3.33299994468689" sequence="EWATYTCVLGFHVYVPVR" >
 				<UserParam type="string" name="my_user_param" value="veryimportant"/>
-                <UserParam type="string" name="target_decoy" value="target"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_2" accession="BSA3" score="0" sequence="AXXXSTFDFYQRRLVTLAESPRAPSX" >
-                <UserParam type="string" name="target_decoy" value="target"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_3" accession="test2_rev" score="0" sequence="EDQSCPSERRRAFSRLRFGPS" >
-                <UserParam type="string" name="target_decoy" value="decoy"/>
+				<UserParam type="string" name="target_decoy" value="decoy"/>
 			</ProteinHit>
 			<ProteinHit id="PH_4" accession="BSA4" score="0" sequence="EDQSCPSERRRAFSRLRXXXX" >
-                <UserParam type="string" name="target_decoy" value="target"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_5" accession="Thyroglo" score="0" sequence="AAAAAEWATYTCVLGFHVYVPVR" >
-                <UserParam type="string" name="target_decoy" value="target"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_6" accession="BSA4_rev" score="0" sequence="EDQSCPSERRRAFSRLRXXXX" >
-                <UserParam type="string" name="target_decoy" value="decoy"/>
+				<UserParam type="string" name="target_decoy" value="decoy"/>
 			</ProteinHit>
 			<ProteinHit id="PH_7" accession="Thyroglo_rev" score="0" sequence="AAAAAEWATYTCVLGFHVYVPVR" >
-                <UserParam type="string" name="target_decoy" value="decoy"/>
+				<UserParam type="string" name="target_decoy" value="decoy"/>
 			</ProteinHit>
 			<ProteinHit id="PH_8" accession="revonly_rev" score="0" sequence="DFIANGER" >
-                <UserParam type="string" name="target_decoy" value="decoy"/>
+				<UserParam type="string" name="target_decoy" value="decoy"/>
 			</ProteinHit>
 		</ProteinIdentification>
 		<PeptideIdentification score_type="Mascot" higher_score_better="true" significance_threshold="31.8621" MZ="0" RT="0" >
-			<PeptideHit score="40" sequence="STFDFYQRRLVTLAESPRAPSP" charge="2" aa_before="X" aa_after="G" protein_refs="PH_0 PH_2" >
+			<PeptideHit score="40" sequence="STFDFYQRRLVTLAESPRAPSP" charge="2" aa_after="G ]" start="227 4"  end="248 25" protein_refs="PH_0 PH_2" >
```

Comment:
There is no aa_before information available (all are unknown) so writing is omitted.
Reading it back in produces the correct result.